### PR TITLE
`Dropout + Add + Layernorm/RMSNorm` benchmarks

### PR DESCRIPTION
**Pull request by Priya2698**

### CODE REVIEW

**Review by wujingyue — APPROVED** [review]

File: python_benchmarks/test_dropout_layernorm_bwd.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import pytest
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
+from .core import run_benchmark, clear_cuda_cache
+import torch
+from .global_params import generate_input_sizes, FLOAT_DTYPES, PROMOTE_DTYPES
+
+
+def dropout_layernorm_bwd_fusion(
```
Comment (jacobhinkle) [inline python_benchmarks/test_dropout_layernorm_bwd.py:9]: For these really big fusions, it would be nice to have a comment or docstring that tells us what is being computed in some more detail.

**Review by jacobhinkle — APPROVED** [review]
LGTM

File: python_benchmarks/test_dropout_layernorm_bwd.py (hunk 1 above)
Comment (Priya2698) [reply python_benchmarks/test_dropout_layernorm_bwd.py:9]: Added a few comments. Will keep in mind for future benchmark PRs.

**Review by liqiangxl — APPROVED** [review]
LGTM.
